app/templates/group.html

```diff
@@ -36,12 +36,14 @@
 						{%- endfor %}
 						</td>
 					</tr>
-					{%- if group.detail.bug_ticket %}
 					<tr>
 						<td>Ticket</td>
-						<td>{{ bug_ticket(group.detail.bug_ticket) }}</td>
+					        {%- if group.detail.bug_ticket %}
+                                                    <td>{{ bug_ticket(group.detail.bug_ticket) }}</td>
+                                                {%-else %}
+                                                    <td><a href="{{ bug_link }}">Create ticket</a></td>
```

Comment:
"Create ticket" should be replace by "Create" otherwise its a tautology as the row itself is about "Ticket".